prophic3/prophic3.cpp

```diff
@@ -571,6 +571,25 @@ bool ProphIC3::check_axioms_over_bmc(TermSet &untimed_axioms,
       }
     }
 
+    if (!violated_axioms.size()) {
```

Comment:
is this done as the last refinement step? (It should be)

Reply:
Yep! It's only added as a last resort.